README.rst

```diff
@@ -17,38 +14,87 @@ https://letsencrypt.org. Be sure to checkout the
 About the Let's Encrypt Client
 ==============================
 
-|build-status| |coverage| |docs| |container|
+The Let's Encrypt Client is a fully-featured, extensible client for the Let's
+Encrypt CA (or any other CA that speaks the `ACME
+<https://github.com/ietf-wg-acme/acme/blob/master/draft-ietf-acme-acme.md>`_
+protocol) that can automate the tasks of obtaining certificates and
+configuring webservers to use them.
+
+Installation
+------------
+
+If ``letsencrypt`` is packaged for your OS, you can install it from there, and
+run it by typing ``letsencrypt``.  Because not all operating systems have
+packages yet, we provide a temporary solution via the ``letsencrypt-auto``
```

Comment:
Should we wait to talk about `letsencrypt-auto` until @erikrose makes his changes? Starting tomorrow, these instructions are out of date.